site/_includes/site-header.html

```diff
@@ -7,7 +7,7 @@
             <li class="home"><a href="/" title="{{ site.title }}">Home</a></li>
             <li class="blog"><a href="/blog" title="Blog Posts">Blog</a></li>
             <li class="community"><a href="/community" title="Community">Community</a></li>
-            <li class="docs"><a href="/docs/v0.15.0">Documentation</a></li>
+            <li class="docs"><a href="/docs">Documentation</a></li>
```

Comment:
Also need to adjust the URL for the footer: https://github.com/heptio/sonobuoy/pull/795/files#diff-0a55ba953b4a00424ce1150c8cb5e4efR17

Reply:
Thanks for catching that, I've updated the PR.